pom.xml

```diff
@@ -23,7 +23,7 @@
   </parent>
 
   <artifactId>google-compute-engine</artifactId>
-  <version>3.2.1-SNAPSHOT</version>
+  <version>3.2.2-SNAPSHOT</version>
```

Comment:
If you're releasing this as 3.2.1 then this should remain as 3.2.1, the maven release plugin will handle the version bump for you.

Reply:
Updated to 3.3.0